### PR TITLE
Fix setup.py with package_data

### PR DESCRIPTION
**Pull request by mrakitin**
This is intended to fix missing *.ui, images, icons, *.txt, etc files.

### CODE REVIEW

File: pycxdgui/startGUI.py [hunk 1]
```diff
@@ -13,7 +15,7 @@
 SDIR = "../storage"
 sxsdesc = "B002.sxs"
 
-configfile = "saxsgui-config.yml"
+configfile = pkg_resources.resource_filename("pycxdgui", "saxsgui-config.yml")
```
Comment (mrakitin) [inline pycxdgui/startGUI.py:18]: This way the installed program will recognize where to look for the config file.

**Comment by mrakitin** [comment]
@jrmlhermitte, I offered a bunch of changes in the modules fixing relative imports. With my current fixes I'm able to run the GUI at SMI using 2 methods (after activation of `collection-2018-1.0` conda env):
- `python -m pycxdgui.startGUI`
- `python /opt/conda_envs/collection-2018-1.0/lib/python3.6/site-packages/pycxdgui/startGUI.py`

Please check the rest of files which I might have missed to update imports, or if you prefer, it would be reasonable to use absolute imports everywhere, e.g. `from pycxdgui.tools.optics import calc_q` rather than sitting in the directory with `tools` and importing it directly such as `from tools.optics import calc_q`. Otherwise Python would not know where to take those `tools`.

**Comment by jrmlhermitte** [comment]
Thanks @mrakitin ! I tested and it works. I had to make some small modifications for the icons to show up.


icons


thanks again!

**Comment by mrakitin** [comment]
Wonderful! I'll try at SMI.

File: pycxdgui/startGUI.py (hunk 1 above)
Comment (jrmlhermitte) [reply pycxdgui/startGUI.py:18]: yeah we might want to search through a list of paths in the future. but for now this is good. thanks!

**Review by jrmlhermitte — COMMENTED** [review]
looks good, works on CMS with pip install from your branch. ill await your test at SMI. thanks @mrakitin !

**Comment by jrmlhermitte** [comment]
ok looks good to me! merging, thanks @mrakitin !

**Comment by mrakitin** [comment]
Thanks for merging! Hopefully we can improve it even more soon :)

**Pull request merged**